presto-docs/src/main/sphinx/admin/dist-sort.rst

```diff
@@ -0,0 +1,17 @@
+================
+Distributed sort
```

Comment:
`redistribute_sort` property and corresponding functionality needs to be described as well